airflow/cli/commands/variable_command.py

```diff
@@ -76,7 +77,24 @@ def variables_import(args):
 
 def variables_export(args):
     """Exports all the variables to the file."""
-    _variable_export_helper(args.file)
+    var_dict = {}
+    with create_session() as session:
+        query = session.query(Variable).all()
+
+        data = json.JSONDecoder()
+        for variable in query:
+            try:
+                value = data.decode(variable.val)
+            except json.decoder.JSONDecodeError:
+                value = variable.val
+            var_dict[variable.key] = value
```

Comment:
```suggestion
        rows = session.query(Variable).all()

        data = json.JSONDecoder()
        for row in rows:
            try:
                value = data.decode(row.val)
            except json.decoder.JSONDecodeError:
                value = row.val
            var_dict[row.key] = value
```

Maybe something like this?

Reply:
I think we can drop the `all()` as well since it’s only iterated once.

Reply:
Apologies for the delay, had been unwell lately. 

Thanks for the inputs. Ive updated the patch.